cassandra_nodetool/tests/conftest.py

```diff
@@ -0,0 +1,104 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import subprocess
+import os
+import logging
+import pytest
+import time
+
+import common
+
+log = logging.getLogger(__file__)
+
+
+def wait_on_docker_logs(container_name, max_wait, sentences):
+    count = 0
+    args = [
+        'docker',
+        'logs',
+        container_name
+    ]
+    out = subprocess.check_output(args)
+    log.info("Waiting for {} to come up".format(container_name))
+    if sentences is not None:
+        while count < max_wait and not any(s in out for s in sentences):
+            time.sleep(1)
+            out = subprocess.check_output(args)
+            count += 1
+
+        if any(s in out for s in sentences):
+            log.info(" {} is up!".format(container_name))
+            return True
+        else:
+            log.info(out)
+            return False
+
+    return False
+
+
+def get_container_ip(container_id_or_name):
+    """
+    Get a docker container's IP address from its id or name
+    """
+    args = [
+        'docker', 'inspect',
+        '-f', '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}', container_id_or_name
+    ]
+
+    return subprocess.check_output(args).rstrip('\r\n')
+
+
+@pytest.fixture(scope="session")
```

Comment:
In order to avoid having to make this a test function argument, let's make this `@pytest.fixture(scope="session", autouse=True)`.